Compile error ('_SC_PAGESIZE' undeclared)

Hello, I checkout commit 38c429a and try build it on mingw+gcc 8.2 but failed with the error `'_SC_PAGESIZE' undeclared`. What might be the problem?

```
FAILED: crypto/CMakeFiles/crypto.dir/compat/getpagesize.c.obj 
/usr/local/base/build/install/bin/x86_64-w64-mingw32-gcc -DCPPFLAGS -DHAVE_ASPRINTF -DHAVE_CLOCK_GETTIME -DHAVE_STRCASECMP -DHAVE_STRNLEN -DLIBRESSL_INTERNAL -DNO_CRYPT -DNO_SYSLOG -DOPENSSLDIR=\"/mnt/c/base/build/install/mingw/etc/ssl\" -DOPENSSL_NO_ASM -DOPENSSL_NO_HW_PADLOCK -DOPENSSL_NO_SPEED -DSIZEOF_TIME_T=8 -DWIN32_LEAN_AND_MEAN -D_CRT_DEPRECATED_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -D_POSIX_THREAD_SAFE_FUNCTIONS -D_REENTRANT -D_WIN32_WINNT=0x0600 -D__BEGIN_HIDDEN_DECLS="" -D__END_HIDDEN_DECLS="" -Drestrict -Icrypto/. -Icrypto/asn1 -Icrypto/bn -Icrypto/dsa -Icrypto/evp -Icrypto/modes -Icrypto/../include/compat -Icrypto/../include -O2 -Wall -O3 -DNDEBUG   -Wno-pointer-sign -MD -MT crypto/CMakeFiles/crypto.dir/compat/getpagesize.c.obj -MF crypto/CMakeFiles/crypto.dir/compat/getpagesize.c.obj.d -o crypto/CMakeFiles/crypto.dir/compat/getpagesize.c.obj   -c crypto/compat/getpagesize.c
crypto/compat/getpagesize.c: In function 'getpagesize':
crypto/compat/getpagesize.c:16:9: warning: implicit declaration of function 'sysconf' [-Wimplicit-function-declaration]
  return sysconf(_SC_PAGESIZE);
         ^~~~~~~
crypto/compat/getpagesize.c:16:17: error: '_SC_PAGESIZE' undeclared (first use in this function)
  return sysconf(_SC_PAGESIZE);
                 ^~~~~~~~~~~~
crypto/compat/getpagesize.c:16:17: note: each undeclared identifier is reported only once for each function it appears in
crypto/compat/getpagesize.c:18:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
```

I also tried compiling tarball releases. All releases on 15 Dec (2.9,2.8.3) gave same error. Tarball v2.8.2 works. What I can observed is it trying to compile `getpagesize.c` while release v2.8.2 doesn't compile this file on mingw platform

EDIT:
Looks like commit 8fee85b is the fault. Undoing this commit, will make it compile sucessfully

--- a/crypto/compat/getpagesize.c
+++ b/crypto/compat/getpagesize.c
@@ -2,13 +2,13 @@
 
 #include <unistd.h>
 
-#ifdef _MSC_VER
+#ifdef _WIN32
 #include <windows.h>
 #endif
 
 int
 getpagesize(void) {
-#ifdef _MSC_VER
+#ifdef _WIN32
 	SYSTEM_INFO system_info;
 	GetSystemInfo(&system_info);
 	return system_info.dwPageSize;

